app/views/TrustStillProcessing.scala.html

```diff
@@ -27,10 +27,9 @@
 @components.heading(
  headingKey = "stillProcessing.heading",
  headingSize = "heading-large govuk-fieldset__heading",
- subheading = Some(messages("stillProcessing.subheading", utr))
+ subheading = Some(subheading(utr))
```

Comment:
Should `utr` be renamed to `identifier` or is this page specific to UTRs?

Reply:
It should on all of them :D